src/main/resources/config/liquibase/changelog/20210910180000_changelog.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.8.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="alexander.malyuk" id="20210910180000">
+        <!-- Introduction of notification option -->
+        <createTable tableName="notification_option">
+            <column name="id" type="BIGINT" autoIncrement="${autoIncrement}">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="user_optionPK"/>
+            </column>
+            <column name="option_specifier" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="webapp" type="BIT(1)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="email" type="BIT(1)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addForeignKeyConstraint
+            baseTableName="notification_option"
+            baseColumnNames="user_id"
+            constraintName="fk_user_option_user_id"
```

Comment:
nit: would fk_notification_option_user_id be more suitable?

Reply:
Yes, it would, this is a big/old PR  problem.  I fear if I change it then e.g. there will be issues on TS3 ...

Reply:
then leave it as it is ;)!